Framework/Dotzu/Dotzu/ManagerListLogViewController.swift

```diff
@@ -81,7 +82,54 @@ class ManagerListLogViewController: UIViewController {
         LogNotificationApp.resetCountBadge.post(Void())
     }
 
-    @IBAction func resetLogs(_ sender: Any) {
+    @IBAction func showActions() {
+
+        let alertController = UIAlertController(title: "More actions",
+                                                message: nil,
+                                                preferredStyle: .actionSheet)
+
+        let sendLogsButton = UIAlertAction(title: "Email logs",
+                                           style: .default,
+                                           handler: { [weak self] (action) in self?.emailLogs() })
+        alertController.addAction(sendLogsButton)
+
+        let resetLogsButton = UIAlertAction(title: "Reset logs & network",
+                                            style: .destructive,
+                                            handler: { [weak self] (action) in self?.resetLogs() })
+        alertController.addAction(resetLogsButton)
+
+        let cancelButton = UIAlertAction(title: "Cancel", style: .cancel, handler:nil)
+        alertController.addAction(cancelButton)
+
+        self.navigationController?.present(alertController, animated: true, completion: nil)
```

Comment:
Why not on the current UIViewController ?

Reply:
true, I missed it. I will update it.